addons/mail/models/update.py

```diff
@@ -98,10 +98,6 @@ def update_notification(self, cron_mode=True):
             # old behavior based on res.log; now on mail.message, that is not necessarily installed
             user = self.env['res.users'].sudo().browse(SUPERUSER_ID)
```

Comment:
You can remove this also.

Reply:
It still uses in the call of `message_post` :cry:

Reply:
Uh, true, I'd missed that.